_posts/2020-04-29-#18038.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: pr
+date: 2020-04-29
+title: "Mempool tracks locally submitted transactions to improve wallet privacy"
+pr: 18038
+authors: [amitiuttarwar]
+components: ["p2p"]
+host: amitiuttarwar
+status: upcoming
+commit:
+---
+
+## Notes
+- Today's PR has been extracted from
+  [#16698](https://github.com/bitcoin/bitcoin/pull/16698), which is a project
+  to improve privacy that involves moving rebroadcast functionality from the
+  wallet to the node. Having a smaller set of changes is always preferable
+  since it makes the PR easier to reason about and review. We discussed #16698
+  at a previous [review club](https://bitcoincore.reviews/16698.html).
+
+- On current master, the wallet has a method called `ResendWalletTransactions`
+  [link](https://github.com/bitcoin/bitcoin/blob/master/src/wallet/wallet.cpp#L1971).
+  This method is intended to _rebroadcast_ transactions, but also serves to
+  ensure a successful initial broadcast. Since only the wallet that originated
+  the txn will rebroadcast a transaction, there is a potential privacy leak
+  everytime `ResendWalletTransactions` runs.
+
+- Today's PR reduces the frequency that `ResendWalletTransactions` runs from
+  ~30 min to every ~day. To do
+  so, we extract the guarantees around initial broadcast.
+
+- Lets clarify some language:
+  - initial broadcast: when a node notifies the network about a transaction
+  - rebroadcast: when a node sends subsequent notifications to the network
+    about that transaction
+  - unbroadcast (introduced in this pr): transactions where the initial
+    broadcast hasn't been deemed successful
+
+- The mempool keeps track of transactions that it attempts to initially
+  broadcast. It uses a heuristic of receiving a relevant `GETDATA` to indicate
+  a successful initial broadcast. Every 10-15 minutes, it re-attempts broadcast
+  of the unbroadcast transactions (what a mouthful!)
```

Comment:
Could make clear whether this paragraph describes things before or after your PR.